### PR TITLE
Implement update quarterly

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

File: src/OpenLoco/Company.h [hunk 1]
```diff
@@ -50,10 +50,15 @@ namespace OpenLoco
...
-            uint8_t pad_00[0x80];
-            uint32_t var_80; // 0x528
-            uint32_t var_84; // 0x52C
-            uint8_t var_88;  // 0x530
+            uint8_t var_00;
+            uint8_t pad_01[0x44 - 0x01];
+            uint8_t var_44; //0x4EC size of var_66
+            uint8_t pad_45[0x66 - 0x45];
+            EntityId_t var_66[11]; //0x50E unsure on size
+            currency32_t var_7C;   //0x524
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Company.h:58]:
```suggestion
            uint8_t var_44; // 0x4EC size of var_66
            uint8_t pad_45[0x66 - 0x45];
            EntityId_t var_66[11]; // 0x50E unsure on size
            currency32_t var_7C;   // 0x524
```

File: src/OpenLoco/Vehicles/VehicleHead.cpp [hunk 2]
```diff
@@ -3277,4 +3278,18 @@ namespace OpenLoco::Vehicles
...
+
+    // 0x004C3BA6
+    currency32_t VehicleHead::calculateRunningCost() const
+    {
+        currency32_t totalRunCost = 0;
+        const Vehicle train(this);
+        for (const auto& car : train.cars)
+        {
+            auto* vehObj = ObjectManager::get<VehicleObject>(car.front->object_id);
+            currency32_t runCost = (vehObj->run_cost_factor * currencyMultiplicationFactor[vehObj->run_cost_index]) / 1024;
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Vehicles/VehicleHead.cpp:3290]: We're probably going to be using `currencyMultiplicationFactor` like this more often. Worth having a function to do this in Economy.cpp? I realise you moved this code, so perhaps outside of the scope of this PR.

**Review by AaronVanGeffen — APPROVED** [review]
Implementation does not diverge from master. Left a few minor comments though.

File: src/OpenLoco/Vehicles/VehicleHead.cpp (hunk 2 above)
Comment (duncanspumpkin) [reply src/OpenLoco/Vehicles/VehicleHead.cpp:3290]: Will do a follow on PR.

**Pull request merged**